assets/stylesheets/_typography.scss

```diff
@@ -87,6 +87,20 @@
   text-transform: none;
 }
 
+h1, h2, h3, h4, h5, h6 {
+  margin: 0;
+  padding: 0;
+  line-height: 1.0;
```

Comment:
Nitpicking here, but I'll consolidate this and then just have the `p` below with the one different attribute. I also list all declarations in alphabetical order.